bundle/tests/scorecard/config.yaml

```diff
@@ -120,13 +120,23 @@ stages:
   - entrypoint:
     - cryostat-scorecard-tests
     - cryostat-report
-    image: quay.io/cryostat/cryostat-operator-scorecard:3.0.0-20240511064726
+    image: quay.io/cryostat/cryostat-operator-scorecard:3.0.0-20240527193829
     labels:
       suite: cryostat
       test: cryostat-report
     storage:
       spec:
         mountPath: {}
+  - entrypoint:
```

Comment:
Maybe not include these scorecard updates?

Reply:
^ https://github.com/cryostatio/cryostat-operator/pull/815#issuecomment-2133983253

Reply:
oh right haha :))